app/routes/people/index.js

```diff
@@ -67,6 +68,28 @@ export default class PeopleIndexRoute extends Route {
         person.positionRoute = 'people.person.positions.minister';
       } else if (person.uri.includes('/mandatarissen/')) {
         person.positionRoute = 'people.person.positions.mandatory';
+      } else if (person.uri.includes('/functionarissen/')) {
+        person.positionRoute = 'people.person.positions.agent';
+      }
+
+      // In the case of functionarissen, a bestuursfunctie is linked to multiple units (OCMW and Gemeente)
+      person.organizations = [];
+      if (Array.isArray(person.organization_id)) {
+        let i = 0;
+        person.organization_id.map((id) => {
```

Comment:
I don't think it should be map here but forEach? Something like this (not tested) seems clearer:

```javascript

        for (let [index, id] of person.organization_id.entries()) {
            person.organizations.push({
            id: id,
            classification: person.organization_classification[index],
          });    
        }

Reply:
Aha, true that I did something weird over there :joy: I'll fix it, thanks for the suggestion :)